src/stores/System.jsx

```diff
@@ -72,9 +73,8 @@ export default class SystemStore {
     reaction(
       () => this.trade.price,
       price => {
-        this.stopPriceTicker();
         if (price.gt(0)) {
-          this.priceTicker = setInterval(async () => {
+          this.stopPriceTicker = asyncInterval(async () => {
```

Comment:
dont you need to check if `this.stopPriceTicker` is already defined and execute it then to avoid running timer in the background?